### DISCUSSION TITLE
Is it possible to prevent the selection of the title and some other things ?

### DISCUSSION THREAD

**Discussion by Speedbox666** [davetcc/tcMenu#336] [Q&A]
Hello,

I discovered this great project and was able to create a menu using the TcMenuDesigner 3.2.0 on an Arduino Mega2560 and an SH1106 128X64 I2C OLED display. However, it seems strange to me that the root menu title can be selected. It is great, when I am in a submenu, selecting the title takes me back to the main menu. Also, is it possible to show the ">>" on the right side of the screen? Lastly, can I manually edit the box size of the title?

Thank you very much!

**Comment by davetcc** [comment]
The root title can have an action attached to it as well. See most examples where we attach a callback to the title being pressed. 

In 3.1 and 3.2 we’ve been tidying up the rendering and I think the >> will be on the right in 3.2 IIRC. 

You can adjust the sizes of anything either by changing the theme settings in code generator or even editing the theme file yourself, it is not overwritten unless you delete it yourself.
